tools/test/examples/examples_lib.py

```diff
@@ -19,7 +19,8 @@
 from tools.export import EXPORTERS
 
 SUPPORTED_TOOLCHAINS = ["ARM", "IAR", "GCC_ARM"]
-SUPPORTED_IDES = ["iar", "uvision", "make_gcc_arm", "make_iar", "make_armc5"]
+SUPPORTED_IDES = [exp for exp in EXPORTERS.keys() if exp != "cmsis" and exp != "zip"]
```

Comment:
Ok, it sounds like these two exporters truly are just strange. In that case the way it's implemented now isn't an issue. :+1: